oracle/price-feeder/oracle/oracle_test.go

```diff
@@ -350,6 +353,218 @@ func (ots *OracleTestSuite) TestPrices() {
 	ots.Require().Equal(sdk.MustNewDecFromStr("1"), prices.AmountOf("uusdt"))
 }
 
+func denomList(names ...string) oracletypes.DenomList {
+	var result oracletypes.DenomList
+	for _, n := range names {
+		result = append(result, oracletypes.Denom{Name: n})
+	}
+	return result
+}
+
+func generateValidatorAddr() string {
+	privKey := ed25519.GenPrivKey()
+	pubKey := privKey.PubKey()
+	return sdk.ValAddress(pubKey.Address().Bytes()).String()
+}
+
+func generateAcctAddr() string {
+	privKey := ed25519.GenPrivKey()
+	pubKey := privKey.PubKey()
+	return sdk.AccAddress(pubKey.Address().Bytes()).String()
+}
+
+func TestTick_Scenarios(t *testing.T) {
```

Comment:
nit, test naming is primarily camelcase, we can just call this `TestTickScenarios`